### PR TITLE
FH-1709 - Fix argument passing for apps create

### PR DESCRIPTION
**Pull request by shannonmpoole**

### CODE REVIEW

**Comment by fheng-build-servers** [comment]
@jasonmadigan @david-martin please review this PR.

File: lib/cmd/common/apps.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ function nguiApps(args, cb) {
     return doRead(projectId, appId, cb);
   }else if (action === 'create'){
     args.shift();
-    return create(args, cb);
```
Comment (david-martin) [inline lib/cmd/common/apps.js]: is there a test that can reproduce the issue?

Comment (shannonmpoole) [inline lib/cmd/common/apps.js]: @david-martin I thought about that but it looks like all the unit tests correctly pass the ```{ _: args }``` format

I think that maybe this needs to be more fleshed out?
https://github.com/feedhenry/fh-fhc/blob/master/test/accept/test_as_cli.js

Comment (david-martin) [inline lib/cmd/common/apps.js]: Yes, testing the cli needs a pattern defined for commands that will make api calls. Existing acceptance tests don't.
Lets get this fix published for now

**Comment by david-martin** [comment]
can you rebase from master and change the version to 2.0.1?

**Comment by shannonmpoole** [comment]
@david-martin rebased and version changed.

**Comment by jasonmadigan** [comment]
Published.

```
+ fh-fhc@2.0.1-67
```

**Pull request merged**